phoenicis-scripts/src/main/java/org/phoenicis/scripts/engine/implementation/PolyglotScriptEngine.java

```diff
@@ -33,16 +35,19 @@
     /**
      * Constructor
      *
+     * @param phoenicisSandbox a Phoenicis Sandbox bean
      * @param language The language name
      * @param options A map of options for the Polyglot context
      */
-    public PolyglotScriptEngine(String language, Map<String, String> options) {
+    public PolyglotScriptEngine(PhoenicisSandbox phoenicisSandbox, String language, Map<String, String> options) {
         super();
+        this.phoenicisSandbox = phoenicisSandbox;
```

Comment:
This should be moved down a few lines to not break the formatting

Reply:
mvn formatter is ok with this

Reply:
I meant it breaks the (my) reading flow :)
I normally have a single line break between `super()` and the later assignments, followed by:
1) the constructor parameter assignments
2) the generated values assignments (often based on constructor parameters)

Reply:
Done